### DISCUSSION TITLE
can I disable /api endpoint completely?

### DISCUSSION THREAD

**Discussion by xinyuliu-cb** [tchiotludo/akhq#1556] [Q&A] [answered]
Hi,

For security purposes, can I disable /api endpoint completely?

Thank you

Xinyu

**Comment by tchiotludo** [comment, marked as answer]
It's not possible since the ui is using /api for all the data from backend, will lead to useless instance

**Reply by xinyuliu-cb** [reply]
kk, I can guard it with Ngnix reverse proxy then. Thank you
